source/fundamentals/connection/network-compression.txt

```diff
@@ -0,0 +1,98 @@
+.. _compression:
+.. _network-compression:
+
+===================
+Network Compression
+===================
+
+The {+driver-long+} provides a connection option to compress messages,
+This reduces the amount of data passed over the network between MongoDB
+and your application.
+
+The driver supports the following algorithms:
+
+1. `Snappy <https://google.github.io/snappy/>`__: available in MongoDB 3.4 and later.
+#. `Zlib <https://zlib.net/>`__: available in MongoDB 3.6 and later.
+#. `Zstandard <https://github.com/facebook/zstd/>`__: available in MongoDB 4.2 and later.
+
+The driver tests against the following versions of these libraries:
+
+- ``{+snappyVersion+}``
+- ``{+zstdVersion+}``
+
+If you specify multiple compression algorithms, the driver selects the
+first one that is supported by the MongoDB instance that the driver is
+connected to.
+
+.. note::
+
+   If your application requires Snappy or Zstandard compression, you must add
+   :ref:`explicit dependencies <compression-dependencies>` for those algorithms.
+
+.. _enable-compression:
+
+Specify Compression Algorithms
+------------------------------
+
+You can enable compression on your connection by specifying the
+algorithms in the following ways:
+
+- Adding the ``compressors`` parameter to your ``ConnectionString`` instance
+- Calling the ``compressorList()`` method from the ``MongoClientSettings`` builder
+
+.. tabs::
+
+   .. tab:: ``ConnectionString``
```

Comment:
is there a reason we're formatting the tab labels? only asking since it's not done on any other pages

Reply:
If it's not done elsewhere, I will remove formatting! agree that it should be everywhere or no where